test/Observable_mockito_test.dart

```diff
@@ -0,0 +1,46 @@
+library wCommon.test;
+
+// Testing imports
+import 'package:unittest/unittest.dart';
+import 'package:mockito/mockito.dart';
+import 'package:unittest/html_enhanced_config.dart';
+
+// Testing dependencies
+import 'dart:async';
+import 'dart:js';
+import 'package:wCommon/wCommon.dart';
+
+class MockJsObject extends Mock {}
+
+main() {
+
+  useHtmlEnhancedConfiguration();
+
+  var observable;
+  var jsObservable;
+
+  group('Observable', () {
+
+    setUp(() {
+      jsObservable = new MockJsObject();
+
+      observable = new Observable(jsObservable);
+    });
+
+    test('it should return a stream for the stream property', () {
+      var stream = observable.stream;
+
+      expect(stream, new isInstanceOf<Stream>());
```

Comment:
I should clarify.  The second argument uses the dart Matchers class.  In this case, isInstanceOf is the matcher, and it is one of the more convoluted ones.  If stream was a boolean, there is a isTrue matcher.